src/app.ts

```diff
@@ -13,13 +14,14 @@ initConnection();
 const app = express();
 app.use(bodyParser.json());
 app.use(cors());
-app.use(passport.initialize()); 
+app.use(passport.initialize());
 
 app.get('/', (req, res) => {
   res.send('ScholarX Backend');
 });
 
 app.use('/api/auth', authRouter);
+app.use('/api/profile', profileRouter);
```

Comment:
I made a change to the api: https://api-scholarx.sefglobal.org/api/me/profile

```suggestion
app.use('/api/me', profileRouter);
```